docker-compose.yml

```diff
@@ -27,6 +27,11 @@ services:
       - redpanda-0:/var/lib/redpanda/data
     networks:
       - redpanda_network
+    healthcheck:
+        test: ["CMD", "curl", "-f", "http://localhost:9644/v1/status/ready"]
```

Comment:
Maybe we can use service name instead of localhost?

Reply:
yeah makes sense updated